model2vec/utils.py

```diff
@@ -49,6 +60,60 @@ def save_pretrained(folder_path: Path, embeddings: np.ndarray, tokenizer: Tokeni
 
     logger.info(f"Saved model to {folder_path}")
 
+    # Optionally create the model card
+    if create_model_card:
+        _create_model_card(folder_path, **kwargs)
+
+
+def _create_model_card(folder_path: Path, **kwargs: Any) -> None:
+    """
+    Create a model card and store it in the specified path.
+
+    :param folder_path: The path where the model card will be stored.
+    :param **kwargs: Additional metadata for the model card (e.g., model_name, base_model, etc.).
+    """
+    folder_path = Path(folder_path)
+    model_name = folder_path.name
+
+    # Generate the model card content
+    model_card = _generate_model_card(model_name, **kwargs)
```

Comment:
I wouldn't mind this function containing the function body of `_generate_model_card`. I think the naming is a bit confusing, so maybe just put the body of `_generate_model_card` here?